content/en/docs/installing-on-linux/cluster-operation/add-edge-nodes.md

```diff
@@ -0,0 +1,151 @@
+
+4. Verify your modification:
+
+   ```bash
+   sudo sysctl -p | grep ip_forward
+   ```
+
+   Expected result:
+
+   ```bash
+   net.ipv4.ip_forward = 1
+   ```
+
+## Create Firewall Rules and Port Forwarding Rules
+
+To make sure edge nodes can successfully talk to your cluster, you must forward ports for outside traffic to get into your network. Specifically, map an external port to the corresponding internal IP address (master node) and port based on the table below. Besides, you also need to create firewall rules to allow traffic to these ports (`10000` to `10004`).
+
+| Fields              | External Ports | Fields                  | Internal Ports |
+| ------------------- | -------------- | ----------------------- | -------------- |
+| `cloudhubPort`      | `10000`        | `cloudhubNodePort`      | `30000`        |
+| `cloudhubQuicPort`  | `10001`        | `cloudhubQuicNodePort`  | `30001`        |
+| `cloudhubHttpsPort` | `10002`        | `cloudhubHttpsNodePort` | `30002`        |
+| `cloudstreamPort`   | `10003`        | `cloudstreamNodePort`   | `30003`        |
+| `tunnelPort`        | `10004`        | `tunnelNodePort`        | `30004`        |
+
+## Add an Edge Node
+
+1. Log in to the console as `admin` and click **Platform** in the top left corner.
+
+2. Select **Cluster Management** and navigate to **Edge Nodes** under **Node Management**.
+
+   {{< notice note >}}
+
+   If you have enabled [multi-cluster management](../../../multicluster-management/), you need to select a cluster first.
+
+   {{</ notice >}} 
+
+3. Click **Add Node**. In the dialog that appears, set a node name and enter an internal IP address of your edge node. Click **Validate** to continue.
+
+   {{< notice note >}} 
+
+   - The internal IP address is only used for inter-node communication and you do not necessarily need to use the actual internal IP address of the edge node. As long as the IP address is successfully validated, you can use it.
+   - It is recommended that you check the box to add the default taint.
+
+   {{</ notice >}} 
+
+4. Copy the command automatically created under **Add Command** and run it on your edge node.
+
+   {{< notice note >}}
+
+   Make sure `wget` is installed on your edge node before you run the command.
+
+   {{</ notice >}} 
+
+   ![edge-node-dialog](/images/docs/installing-on-linux/add-and-delete-nodes/add-edge-nodes/edge-node-dialog.png)
+
+5. Close the dialog, refresh the page, and the edge node will appear in the list.
+
+   ![edge-node-added](/images/docs/installing-on-linux/add-and-delete-nodes/add-edge-nodes/edge-node-added.png)
+
+   {{< notice note >}}
+
+   After an edge node is added, if you cannot see CPU and memory resource usage on the **Edge Nodes** page, make sure [Metrics Server](../../../pluggable-components/metrics-server/) 0.4.1 or later is installed in your cluster.
+
+   {{</ notice >}}
+
+## Remove an Edge Node
+
+1. On your edge node, run the following commands:
+
+   ```bash
+   ./keadm reset
```

Comment:
Add another point, it is recommended that all  the workloads or deployments  on the edge 
 removed before executing `keadm reset`.

Reply:
Added.